pkgs/by-name/ti/tidal-hifi/electron.nix

```diff
@@ -0,0 +1,30 @@
+{
+  fetchzip,
+  lib,
+}:
+let
+  /*
+    see:
+    https://github.com/Mastermindzh/tidal-hifi/blob/master/build/electron-builder.base.yml
+     for the expected version
+  */
+  version = "35.1.1";
+in
+(fetchzip {
+  url = "https://github.com/castlabs/electron-releases/releases/download/v${version}+wvcus/electron-v${version}+wvcus-linux-x64.zip";
+  hash = "sha256-AkPKeG7MrCBlk41qXZxFPRukUPRcIUanq6fJPx5d3RU=";
+  stripRoot = false;
+
+}).overrideAttrs
+  (
+    final: _: {
+      inherit version;
+      pname = "castlabs-electron";
```

Comment:
I think you would also need to override `name` if you want this to show up in the error message. Right now it says

```
Package ‘source’ in nixpkgs/pkgs/by-name/ti/tidal-hifi/electron.nix:21 has an unfree license (‘unfreeRedistributable’), refusing to evaluate.
```